cmd/podman/containers/export.go

```diff
@@ -79,7 +79,7 @@ func export(cmd *cobra.Command, args []string) error {
 		if err := parse.ValidateFileName(outputFile); err != nil {
 			return err
 		}
-		// open file here with WRONLY since on MacOS it can fail to open /dev/stderr in read mode for example
+		// open file here with WRONGLY since on MacOS it can fail to open /dev/stderr in read mode for example
```

Comment:
This was correct, the option is called `WRONLY`, alternatively change it to `O_WRONLY`